react-native-libraries.json

```
@@ -8590,5 +8590,18 @@
     "ios": true,
     "android": true,
     "newArchitecture": true
+  },
+  {
+    "githubUrl": "https://github.com/wix/Detox",
+    "npmPkg": "detox",
+    "nameOverride": "Detox",
+    "examples": [
+      "https://github.com/wix/Detox/tree/master/examples"
```

```suggestion
      "https://github.com/wix/Detox/tree/master/examples/demo-react-native",
      "https://github.com/wix/Detox/tree/master/examples/demo-react-native-detox-instruments",
      "https://github.com/wix/Detox/tree/master/examples/demo-react-native-jest"
```

Let's list individual example apps instead.

---

To be honest, we have plans to delete some of the projects in the list, and replace them with new ones.
I replaced the folder name with a specific app, and will update this list once when we'll have the new projects.